src/main/java/edu/kis/powp/command/OperateToCommand.java

```diff
@@ -1,15 +1,19 @@
 package edu.kis.powp.command;
 
+import edu.kis.powp.jobs2d.Job2dDriver;
+
 public class OperateToCommand implements DriverCommand {
-    private int x;
-    private int y;
+    private final int x;
+    private final int y;
+    private final Job2dDriver driver;
 
-    OperateToCommand(int x, int y) {
+    public OperateToCommand(int x, int y, Job2dDriver driver) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).